Backend/app/dashboard/Lib/src/Tools/Auth/Dashboard.hs

```diff
@@ -93,3 +93,5 @@ verifyDashboardAccess requiredAccessType personId = do
         else throwError AccessDenied
     DRole.DASHBOARD_USER ->
       pure person.id
+    DRole.FLEET_OWNER ->
```

Comment:
`DashboardAuth FLEET_OWNER` will success for each dashboard user (the same as `DashboardAuth DASHBOARD_USER`). So, adding new access type `FLEET_OWNER` make no sense, I think

Reply:
ok so here we can use same as done for admin .